CMakeLists.txt

```diff
@@ -65,6 +65,12 @@ IF (UNIX)
     )
 ENDIF (UNIX)
 
+# Usage of RPATH for macOS is enabled by default; this just suppresses the CMP0042 warning
+# when building libwildmidi_dynamic.
+if (POLICY CMP0042)
```

Comment:
Minor style nit: uppercase `IF` is used elsewhere in this file.

Reply:
Oh, I intended to use uppercase IF. I experimented with a few things that used `IF` but I guess this one didn't. Fixed.